Configurations/windows-makefile.tmpl

```diff
@@ -586,12 +586,10 @@ reconfigure reconf:
 
       if ($args{src} =~ /\.html$/) {
 	  my $title = basename($args{src}, ".html");
+	  my $pod = $args{generator}->[0]
 	  return <<"EOF";
-$args{src}: $args{generator}->[0]
-	pod2html "--podroot=\$(SRCDIR)/doc" --htmldir=.. \\
-		 --podpath=man1:man3:man5:man7 "--infile=\$?" "--title=$title" \\
-	| \$(PERL) -pe ^"s^|href=\\^"http://man\\.he\\.net/^(man\\d/[^^\\^"]+^)^(?:\.html^)?^"^|href=\\^"../\$\$1.html^|g;^" \\
-	> \$\@
+$args{src}: "$pod"
+	\$(PERL) \$(SRCDIR)/util/mkpod2html.pl -i "$pod" -o \$\@ -t "$title" -r "\$(SRCDIR)/doc"
```

Comment:
Note that it's different on Windows.  With `nmake`, quotes *do* have a meaning and are not taken literally in the same as on Unix.

Reply:
rodger.